dojo/api_v2/views.py

```diff
@@ -192,6 +192,19 @@ def get_serializer_class(self):
         else:
             return serializers.FindingSerializer
 
+    @action(detail=True, methods=['get'])
```

Comment:
The `GET` method that actually induce a change seems weird to me. In my mind, `GET` should trigger no change.

Would it better to eventually call `PUT` or `PATCH` instead of `GET`?